tests/integration/update.bats

```diff
@@ -69,7 +69,7 @@ EOF
         MEM_SWAP="memory.memsw.limit_in_bytes"
         SD_MEM_SWAP="unsupported"
         SYSTEM_MEM=$(cat "${CGROUP_MEMORY_BASE_PATH}/${MEM_LIMIT}")
-        SYSTEM_MEM_SWAP=$(cat "${CGROUP_MEMORY_BASE_PATH}/$MEM_SWAP")
+        SYSTEM_MEM_SWAP=$(cat "${CGROUP_MEMORY_BASE_PATH}/$MEM_SWAP" || true)
```

Comment:
test -f

Reply:
This was my first version, but then I realized I have to repeat the file name twice and this is not actually needed since we can just ignore the failure here.

Reply:
OK, I can do without repeating the file name. Please see updated.